clifford/_mvarray.py

```diff
@@ -29,7 +53,8 @@ def value(self):
         """
         Return an np array of the values of multivectors
         """
-        return np.array([mv.value for mv in self])
+        v_value_get = np.vectorize(lambda x: x.value, otypes=[np.ndarray], signature='()->(n)')
```

Comment:
You should vectorize this in the constructor.

Also, this `otype` is wrong - can you use `float`? to avoid the cast below?

Reply:
Better than `float` would be to use the `dtype` returned by `_interrogate_nested_mvs`

Reply:
agreed on the dtype, but I was struggling to work out how to get hold of it properly in the MVArray instance. __new__ appears to operate on the class `cls`... can I just write `cls.dtype = dtype` or will that modify it for all instances of MVArray?

Reply:
`self` is the return value of `.view` - you can attach properties to that object.

Don't use `.dtype` though, as that's managed by numpy!